app/extensions/edm/__init__.py

```diff
@@ -33,6 +33,9 @@ class EDMManager(RestManager):
     NAME = 'EDM'
     ENDPOINT_PREFIX = 'api'
 
+    # this is based on edm date of most recent commit (we must be at or greater than this)
+    VERSION_REQUIRED = '2021-04-28 14:00:00 -0700'
```

Comment:
Name is wrong. It should be MIN_VERSION, VERSION_REQUIRED implies that it must be exactly that version